src/PrestaShopBundle/Controller/Admin/Sell/Catalog/DiscountController.php

```diff
@@ -235,6 +243,78 @@ public function deleteAction(int $discountId): RedirectResponse
     private function getErrorMessages(Exception $e): array
     {
         return [
+            DiscountNotFoundException::class => $this->trans('The object cannot be loaded (or found).', [], 'Admin.Notifications.Error'),
+            // todo: use more specific message for constraint exceptions
+            DiscountConstraintException::class => $e->getMessage(),
+            CannotUpdateDiscountException::class => $this->trans('An error occurred while updating the discount.', [], 'Admin.Notifications.Error'),
         ];
     }
+
+    /**
+     * @param Request $request
+     *
+     * @return int[]
+     */
+    private function getBulkDiscountsFromRequest(Request $request): array
+    {
+        return array_map('intval', $request->request->all('discount_bulk'));
+    }
+
+    /**
+     * Process bulk action for discount status enabling.
+     *
+     * @param Request $request
+     *
+     * @return RedirectResponse
+     */
+    #[DemoRestricted(redirectRoute: 'admin_discounts_index')]
+    #[AdminSecurity("is_granted('update', request.get('_legacy_controller')) && is_granted('create', request.get('_legacy_controller')) && is_granted('delete', request.get('_legacy_controller'))", redirectRoute: 'admin_discounts_index', message: 'You do not have permission to update this.')]
+    public function bulkEnableStatusAction(Request $request): RedirectResponse
+    {
+        try {
+            $discountIds = $this->getBulkDiscountsFromRequest($request);
+
+            $command = new BulkEnableDiscountsCommand($discountIds);
```

Comment:
```suggestion
            $command = new BulkUpdateStatusDiscountsCommand($discountIds, true);
```
If we only keep one command, it doesn't mean we can't have two controllers for enable/disable

Reply:
Yep, if we keep only one command. So what do we do? :D